src/Components/Search/Suggestions/SuggestionItem.tsx

```diff
@@ -113,32 +119,66 @@ const QuickNavigation: React.FC<{
   showArtworksButton: boolean
   showAuctionResultsButton: boolean
 }> = ({ href, showArtworksButton, showAuctionResultsButton }) => {
+  const { trackEvent } = useTracking()
   if (!showArtworksButton && !showAuctionResultsButton) return null
 
   return (
     <Flex flexWrap="wrap">
       {!!showArtworksButton && (
-        <QuickNavigationItem to={`${href}/works-for-sale`} Icon={ArtworkIcon}>
+        <QuickNavigationItem
+          handleClick={() => {
+            const trackingEvent: SearchSuggestionQuickNavigationItemSelected = {
+              context_module: ContextModule.header,
+              destination_path: `${href}/works-for-sale`,
+              action: ActionType.searchSuggestionQuickNavigationItemSelected,
+              label: "Artworks",
+            }
+
+            trackEvent(trackingEvent)
+          }}
+          to={`${href}/works-for-sale`}
+          Icon={ArtworkIcon}
+        >
           Artworks
         </QuickNavigationItem>
       )}
       {!!showAuctionResultsButton && (
-        <QuickNavigationItem to={`${href}/auction-results`} Icon={AuctionIcon}>
+        <QuickNavigationItem
+          handleClick={() => {
+            const trackingEvent: SearchSuggestionQuickNavigationItemSelected = {
+              context_module: ContextModule.header,
+              destination_path: `${href}/auction-results`,
+              action: ActionType.searchSuggestionQuickNavigationItemSelected,
+              label: "Auction Results",
+            }
+
+            trackEvent(trackingEvent)
+          }}
+          to={`${href}/auction-results`}
+          Icon={AuctionIcon}
+        >
           Auction Results
         </QuickNavigationItem>
       )}
     </Flex>
   )
 }
 
-const QuickNavigationItem: React.FC<{ to: string } & PillProps> = ({
+interface QuickNavigationItemProps {
+  to: string
+  handleClick?: (event: React.MouseEvent<HTMLAnchorElement>) => void
```

Comment:
**suggestion**: rename to `onClick` prop

Reply:
i think yep (more matches with `handleClick` name)

| `onClick` | After |
| ------------- | ------------- |
| <img width="504" alt="1" src="https://user-images.githubusercontent.com/3513494/223185359-61312e61-b720-4f96-b190-08f0060ef97a.png"> | <img width="504" alt="2" src="https://user-images.githubusercontent.com/3513494/223185393-bcb3d534-3994-4c0f-a595-b6c7eb9fdc31.png"> |